app/models/user.rb

```diff
@@ -12,4 +21,41 @@ class User < ActiveRecord::Base
   def has_authority?
     authority == "admin" || authority == "mod"
   end
+
+  def karma_score
+    reviews = Review.where(user_id: id)
+    games = Game.where(user_id: id)
+    xp = 0
+    reviews.each do |review|
+      xp += 1
+      xp += review.vote_count
+    end
+    games.each do |game|
```

Comment:
Unused block argument - `game`. You can omit the argument if you don't care about it.